pkg/kubectl/cmd/get/get.go

```diff
@@ -733,7 +733,7 @@ func (o *GetOptions) printGeneric(r *resource.Result) error {
 	}
 
 	var obj runtime.Object
-	if !singleItemImplied || len(infos) > 1 {
+	if !singleItemImplied || len(infos) > 1 || len(infos) == 0 {
```

Comment:
simplify `len(infos) > 1 || len(infos) == 0` to `len(infos) != 1`?

Reply:
true